builder/hcloud/step_create_server.go

```diff
@@ -59,13 +59,23 @@ func (s *stepCreateServer) Run(ctx context.Context, state multistep.StateBag) mu
 		if err != nil {
 			return errorHandler(state, ui, "Could not find image", err)
 		}
+		if image == nil {
+			return errorHandler(state, ui, "", fmt.Errorf("Could not find image"))
+		}
```

Comment:
Can you add a test case to ensure that this does not regress?

Reply:
Done